src/flint/qadic.jl

```diff
@@ -783,12 +783,17 @@ function (R::FlintQadicField)(n::fmpz_poly)
 end
 
 function (R::FlintQadicField)(n::fmpq_poly)
+
+   if degree(n) > degree(R)+1
```

Comment:
Spaces before and after +.

Also, can we just raise the error, "Polynomial larger than field degree". Or do you really expect this to do something meaningful? If so, then please open a ticket.

Reply:
My thought was that the original meaning of this function is that "x" can be viewed as an alias for the generator of Qq over Qp, and thus Qq(x^100) means Qq(x)^100. I'm not attached to this particular logic, so just raising an error is fine as well.

Reply:
I would just raise an error for now. Once you have pushed a commit to fix this and other issues raised, I will merge this.

Reply:
OK. This should be on the most recent commit, along with all the other requested changes. Please let me know if there are problems.